sites/public/pages/applications/review/confirmation.tsx

```diff
@@ -72,27 +74,31 @@ export default () => {
           )}
         </div>
 
-        <div className="form-card__group">
-          <h3 className="form-card__paragraph-title">
-            {t("application.review.confirmation.createAccountTitle")}
-          </h3>
+        {initialStateLoaded && !profile && (
+          <div className="form-card__group">
+            <h3 className="form-card__paragraph-title">
+              {t("application.review.confirmation.createAccountTitle")}
+            </h3>
 
-          <p className="field-note mt-1">
-            {t("application.review.confirmation.createAccountParagraph")}
-          </p>
-        </div>
+            <p className="field-note mt-1">
+              {t("application.review.confirmation.createAccountParagraph")}
+            </p>
+          </div>
+        )}
 
         <div className="form-card__pager">
-          <div className="form-card__pager-row primary">
-            <Button
-              styleType={AppearanceStyleType.primary}
-              onClick={() => {
-                void router.push("/create-account").then(() => window.scrollTo(0, 0))
-              }}
-            >
-              {t("account.createAccount")}
-            </Button>
-          </div>
+          {initialStateLoaded && !profile && (
```

Comment:
Just curious what this change is for?

Reply:
It should only show an option to create a new account if the logged-in state has loaded and the user _isn't_ logged in.